floor/writer_test.go

```diff
@@ -568,12 +567,12 @@ func (r *marshTestRecord) MarshalParquet(obj interfaces.MarshalObject) error {
 		grp.AddField("quux").SetInt64(b.quux)
 	}
 
-	fmt.Printf("marshal data: %s", spew.Sdump(obj.GetData()))
+	//fmt.Printf("marshal data: %s", spew.Sdump(obj.GetData()))
 	return nil
 }
 
 func (r *marshTestRecord) UnmarshalParquet(obj interfaces.UnmarshalObject) error {
-	fmt.Printf("unmarshal data: %s", spew.Sdump(obj.GetData()))
+	//fmt.Printf("unmarshal data: %s", spew.Sdump(obj.GetData()))
```

Comment:
do we need to comment these log statements out?

Reply:
Removed.